app/controllers/wokes_controller.rb

```diff
@@ -0,0 +1,63 @@
+class WokesController < ObjectsController
```

Comment:
The pattern here is similar to any other Active Record controller (even though using form objects).